### PR TITLE
DOC: add README for PDF generation

### PR DESCRIPTION
**Pull request by lacabra**
As per [this comment](https://github.com/unicef/publicgoods-toolkit/issues/65#issuecomment-985038992), I am hereby adding a README to the `pdf/` folder to document how to create the PDF from the Markdown files (and installing the necessary dependencies). It's admittedly very basic, but that's all the information I would need to accomplish this task.

### CODE REVIEW

File: pdf/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# PDF Generation of Toolkit
+
+The contents of this folder are used towards the automated generation of a PDF of the Digital Public Goods Toolkit. This is accomplished through the preprocessing and subsequent collating of selected Markdown files to output a single PDF file.
+
+## Requirements
+
+Generating a PDF, requires both `pandoc` (v2.9 or higher) and `TeX` to be installed in the system.
```
Comment (kfogel) [inline pdf/README.md:7]: No need for a comma in this sentence, FWIW.

File: pdf/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+## Generate the PDF
+
+Simply run `make` inside this folder:
+
+```bash
+cd pdf
+make
+```
+
+which will output a file named `publicgoods-toolkit-{version}.pdf`.
```
Comment (kfogel) [inline pdf/README.md:27]: The new version of the Makefile I just reviewed (in PR #70) no longer puts the "`-{version}`" into the filename, I think?  If so, this line of documentation would need to be adjusted accordingly.

File: pdf/README.md (hunk 1 above)
Comment (lacabra) [reply pdf/README.md]: Thanks, corrected in fc80119

File: pdf/README.md (hunk 2 above)
Comment (lacabra) [reply pdf/README.md]: Correct, updated documentation accordingly in 7c22f79

**Review by kfogel — APPROVED** [review]
Very much looking forward to this being merged!  Thanks for improving & documenting the build process, @lacabra .

**Pull request merged**